src/network/network.cpp

```diff
@@ -1100,13 +1100,53 @@ void NetworkShutDown()
 	NetworkCoreShutdown();
 }
 
+/**
+ * Get the network version string used by this build.
+ * The returned string is guaranteed to be at most NETWORK_REVISON_LENGTH bytes.
+ */
+const char * GetNetworkRevisionString()
+{
+	/* This will be allocated on heap and never free'd, but only once so not a "real" leak. */
+	static char *network_revision = nullptr;
```

Comment:
some scattered uses of nullptr. I know we should be using it, but probably should be consistent until someone bothers to s/NULL/nullptr/

Reply:
My thought was to use nullptr in new code, and just slowly convert as different parts are touched.